### DISCUSSION TITLE
I don't know how to read gltf's Emissive and AO textures.

### DISCUSSION THREAD

**Discussion by btrid** [assimp/assimp#4200] [Q&A] [answered]
I am implementing this project with assimp.
https://github.com/SaschaWillems/Vulkan-glTF-PBR
I don't know how to read Emissive and AO textures.

using model : https://github.com/KhronosGroup/glTF-Sample-Models/tree/master/2.0/DamagedHelmet/glTF
I can read it using tinygltf.
![image](https://user-images.githubusercontent.com/2136466/142989299-59d7ab13-5a39-48a6-a57c-22682bbd4772.png)

**Comment by btrid** [comment, marked as answer]
I found that I should use aiTextureType_EMISSIVE instead of aiTextureType_EMISSION_COLOR.
I haven't taken Ambient Occlusion, but I solved it because I don't need it.

**Reply by Ligh7ninG1118** [reply]
For anyone else stumbled upon this problem, use aiTextureType_LIGHTMAP for your AO texture.
